utilities/cli/tests/test_cli.py

```diff
@@ -218,9 +237,13 @@ def test_lint_fix_command(self):
         args = self.cli.parser.parse_args("lint --fix".split())
         try:
             args.func(args)
+        except FileNotFoundError as e:
+            if not is_ruff_available():
+                self.assertIn("ruff", str(e))  # if not installed, it complains about ruff
```

Comment:
We should emit the error if ruff is available and `FileNotFoundError` occurs:

```suggestion
            if is_ruff_available():
                raise e
            else:
                self.assertIn("ruff", str(e))  # if not installed, it complains about ruff
```